src/ci/resources/enable_metadata_archiving.inc.conf

```diff
@@ -0,0 +1,25 @@
+# To use this partial config to activate archiving, include
+# it at the top of another config file (eg papi_v2_shared_application.inc.conf)
```

Comment:
Note to reviewers: this file is not currently included in any other conf file to avoid interrupting test cases, but allows you to "test along at home" with only a single line addition to the conf file of your choice